src/registrar/utility/csv_export.py

```diff
@@ -782,30 +708,325 @@ def export_data_unmanaged_domains_to_csv(csv_file, start_date, end_date):
+        annotations = {
+            "creator_approved_domains_count": DomainRequestExport.get_creator_approved_domains_count_query(),
+            "creator_active_requests_count": DomainRequestExport.get_creator_active_requests_count_query(),
+            "all_current_websites": StringAgg("current_websites__website", delimiter=delimiter, distinct=True),
+            "all_alternative_domains": StringAgg("alternative_domains__website", delimiter=delimiter, distinct=True),
+            # Coerce the other contacts object to "{first_name} {last_name} {email}"
+            "all_other_contacts": StringAgg(
+                Concat(
+                    "other_contacts__first_name",
+                    Value(" "),
+                    "other_contacts__last_name",
+                    Value(" "),
+                    "other_contacts__email",
+                ),
+                delimiter=delimiter,
+                distinct=True,
+            ),
+        }
+
+        # .values can't go two levels deep (just returns the id) - so we have to include this.
+        additional_values = [
+            "requested_domain__name",
+            "federal_agency__agency",
+            "authorizing_official__first_name",
+            "authorizing_official__last_name",
+            "authorizing_official__email",
+            "authorizing_official__title",
+            "creator__first_name",
+            "creator__last_name",
+            "creator__email",
+            "investigator__email",
+        ]
+
+        # Convert the domain request queryset to a dictionary (including annotated fields)
+        annotated_requests = cls.annotate_and_retrieve_fields(requests, annotations, additional_values)
+        requests_dict = convert_queryset_to_dict(annotated_requests, is_model=False)
+
+        # Write the csv file
+        cls.write_csv_for_requests(writer, cls.all_columns, requests_dict)
+
+    @staticmethod
+    def write_csv_for_requests(
+        writer,
+        columns,
+        requests_dict,
+        should_write_header=True,
+    ):
+        """Receives params from the parent methods and outputs a CSV with filtered and sorted requests.
+        Works with write_header as long as the same writer object is passed."""
+
+        rows = []
+        for request in requests_dict.values():
+            try:
+                row = DomainRequestExport.parse_row_for_requests(columns, request)
+                rows.append(row)
+            except ValueError as err:
+                logger.error(f"csv_export -> Error when parsing row: {err}")
+                continue
+
+        if should_write_header:
+            write_header(writer, columns)
+
+        writer.writerows(rows)
+
+    @staticmethod
+    def parse_row_for_requests(columns, request):
+        """
+        Given a set of columns and a request dictionary, generate a new row from cleaned column data.
+        """
+
+        # Handle the federal_type field. Defaults to the wrong format.
```

Comment:
Do we want to add tests and skip those particular fields if they're not needed for the report?